lib/TokenManager.ts

```diff
@@ -391,8 +391,8 @@ export class TokenManager implements TokenManagerInterface {
   // TODO: renew method should take no param, change in the next major version OKTA-407224
   renew(key): Promise<Token> {
     // Multiple callers may receive the same promise. They will all resolve or reject from the same request.
-    if (this.state.renewPromise) {
-      return this.state.renewPromise;
+    if (this.state.renewPromise[key]) {
```

Comment:
this seems to be fixing a different issue than the unhandled rejection. 
Could you please add a separate line in the changelog for this bugfix and reference the other issue ID if it exists?